src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -864,4 +866,12 @@ private void discoverFreeBlocksAndPos()
             }
         }
     }
+
+    /**
+     * Inits compats
+     */
+    private void discoverModCompat()
+    {
+        ModList.get().isLoaded("resourcefulbees") ? Compatibility.beeHiveCompat = new ResourcefulBeesCompat() : Compatibility.beeHiveCompat = new IBeeHiveCompat() {};
```

Comment:
Wrong code style. Assignment style is used in Minecolonies.

Reply:
Not sure what this means, exactly.

Reply:
`Compatibility.beeHiveCompat = ModList.get().isLoaded("resourcefulbees") ?  new ResourcefulBeesCompat() : new IBeeHiveCompat() {};`
That

Reply:
Ah. I didn't write that particular line, but I did expand that out anyway. Should be okay now, then.